From 6f11e90d488e241a3d735d5e2d9f01bb9e24213c Mon Sep 17 00:00:00 2001
From: Rakshit Gondwal <rakshit.gondwal@chainguard.dev>
Date: Thu, 6 Mar 2025 14:15:53 +0530
Subject: [PATCH] fix: go version in go.mod

The go-fips pipeline is running on go 1.23, while the syft upstream requires go version to be >= 1.24 which results in an error. This patch sets the go version in the go.mod to 1.23

Signed-off-by: Rakshit Gondwal <rakshit.gondwal@chainguard.dev>
---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 9b91e45f..fa410f17 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/anchore/syft
 
-go 1.24.0
+go 1.23.0
 
 require (
 	github.com/CycloneDX/cyclonedx-go v0.9.2
-- 
2.39.5 (Apple Git-154)

